iree/compiler/Codegen/Common/OptimizeVectorTransferPass.cpp

```diff
@@ -93,22 +93,34 @@ struct OptimizeVectorTransferPass
     // Generate vector.shape_cast for dropping leading one dimensions in vector
     // ops. This increases the chance that we can forward more transfer writes
     // to transfer reads.
-    RewritePatternSet patterns(&getContext());
-    mlir::vector::populateVectorTransferDropUnitDimsPatterns(patterns);
-    mlir::vector::populateFlattenVectorTransferPatterns(patterns);
-    mlir::vector::populateCastAwayVectorLeadingOneDimPatterns(patterns);
-    patterns.add<TransposeUnitDimToShapeCast>(&getContext());
-    mlir::vector::populateVectorTransferCollapseInnerMostContiguousDimsPatterns(
-        patterns);
-    if (failed(applyPatternsAndFoldGreedily(funcOp, std::move(patterns)))) {
-      return signalPassFailure();
+    {
```

Comment:
Do we want to just use the lowering of vector ops that exists with the Sandbox drivers?

Reply:
I'm not sure this does what Benoit wants, I haven't been following if this is in the sandbox. Those patterns were added by https://github.com/google/iree/pull/8114 for performance reason. Does the sandbox do that?

Reply:
Thanks for the context.